contracts/deployables/freeze-voting/BaseFreezeVotingV1.sol

```diff
@@ -65,31 +61,54 @@ abstract contract BaseFreezeVotingV1 is
         return _freezeProposalPeriod;
     }
 
-    function freezeProposalVoteCount()
+    function freezePeriod() external view virtual override returns (uint32) {
+        return _freezePeriod;
+    }
+
+    function freezeVotesThreshold()
         external
         view
         virtual
         override
         returns (uint256)
     {
-        return _freezeProposalVoteCount;
+        return _freezeVotesThreshold;
     }
 
-    function userHasFreezeVoted(
-        address user,
-        uint48 proposalId
-    ) external view virtual override returns (bool) {
-        return _userHasFreezeVoted[user][proposalId];
+    function freezeActivated() external view virtual override returns (uint48) {
+        return _freezeActivated;
     }
 
     function isFrozen() external view virtual override returns (bool) {
         return
             _freezeProposalVoteCount >= _freezeVotesThreshold &&
-            block.timestamp < _freezeProposalCreated + _freezePeriod;
+            block.timestamp < _freezeActivated + _freezePeriod;
+    }
+
+    function initializeFreezeVote() internal virtual {
+        _freezeProposalCreated = uint48(block.timestamp - 1);
```

Comment:
Do we know why we had the " - 1" here before?

Reply:
i don't really know tbh